.github/workflows/release.yml

```
@@ -36,3 +48,23 @@ jobs:
       - run: npm publish --access=public --workspace=packages/core
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+  npm-cli:
+    name: npm publish @web3-storage/content-claims-cli
+    needs: release
+    if: force_release_cli || contains(fromJson(needs.release.outputs.paths_released), 'packages/cli')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2.2.3
+        with:
+          version: 8
+      - uses: actions/setup-node@v3
```

let's update some deps while we're here

```suggestion
      - uses: actions/checkout@v4
      - uses: pnpm/action-setup@v2.4.0
        with:
          version: 8
      - uses: actions/setup-node@v4
```